packages/ui/atomic/create-atomic-component-project/template/src/components/results-manager/results-manager.tsx

```diff
@@ -0,0 +1,35 @@
+import {Component, h} from '@stencil/core';
+import blogTemplate from './templates/blog-template.html';
+import youtubeTemplate from './templates/youtube-template.html';
+import chatterTemplate from './templates/chatter-template.html';
+
+import defaultTemplate from './templates/default-template.html';
+
+/**
+ * Component used to manage results & result templates.
+ * See https://docs.coveo.com/en/atomic/latest/usage/create-a-result-list/
+ */
+@Component({
+  tag: 'results-manager',
+  shadow: true,
+})
+export class ResultsManager {
+  public render() {
+    return (
+      <atomic-folded-result-list display="list" image-size="small">
+        <atomic-result-template must-match-source="Youtube">
+          <template innerHTML={youtubeTemplate}></template>
```

Comment:
In case you decide that you prefer to keep the templates.
```suggestion
          {/* deepcode ignore ReactSetInnerHtml: static */}
          <template innerHTML={youtubeTemplate}></template>
```